### ISSUE TITLE
Improve scroll arrow button

### ISSUE THREAD

**Issue by vertigo220** [Droid-ify/client#314]
**Is your feature request related to a problem? Please describe.**
The new arrow button to scroll to the top blocks the version number of the bottom app in the list, which prevents seeing the color-coding to see if the app is installed and updated.

**Describe the solution you'd like**
One possible solution would be to move it over some. Another would be to change its appearance. It could be made transparent, but it's green and would therefore still be problematic for obvious reasons looking at the example screenshot. Perhaps it could just be a green arrow without the square "box" around it, or the box could be an outline without the fill. Or it could only show when scrolling, so when the list is at rest it would auto-hide, at least when at the end of the list as that's the only time it's really an issue since the bottom item can't be scrolled up and out of the way.

**Describe alternatives you've considered**
Instead of using the arrow button, a scroll bar with quick-scroll (grab and drag the bar) could be implemented instead, which would solve this issue and #311 and #195 all at once.

**Additional context**
![Screenshot_20230320_152601_Droid-ify](https://user-images.githubusercontent.com/12024557/226521688-03ef8bd5-783e-42a2-b2e0-137da24b2743.jpg)

![Screenshot_20230320_152554_Droid-ify](https://user-images.githubusercontent.com/12024557/226521714-58d087df-79f6-44be-b4cc-c4b2637f4b0d.jpg)

**Comment by DJCrashdummy** [comment]
additionally it would somehow also close #1332.

**Comment by kaoneko** [comment]
The usual solution is to "add an empty row" below the last item, so the last item in a list is not partially covered by the floating UI element when one has scrolled to the end.

**Comment by Iamlooker** [comment]
@kaoneko we do add empty space below last items to fix this issue, but I think the issue report is to somehow show versions above the arrow which is not good ux/ui

**Comment by vertigo220** [comment]
> [@kaoneko](https://github.com/kaoneko) we do add empty space below last items to fix this issue, but I think the issue report is to somehow show versions above the arrow which is not good ux/ui

No, that's not what I was asking for. As you can see in the screenshot, there is no empty space at the bottom. You can't really tell, because there's no indication if it's scrolled all the way down, and since it's been over 3 years I can't say for sure that it was, but I'm pretty sure it was. There is, however, an empty space now, so I'm guessing it was added at some point between then and now and just wasn't associated with this issue. I think this can be closed, though I still think a scroll bar that can be dragged for quick scrolling would be better, for this type of issue as well as resolving one or two of the other issues mentioned here, and as just generally better UI.

**Comment by DJCrashdummy** [comment]
as already kind of mentioned at one of the related issues (https://github.com/Droid-ify/client/issues/1332#issuecomment-5291036784):

the best would be IMHO to make it possible to take **an indicator** on the right side and by **dragging** it completely **up or down**, move to the top or bottom of the list.
such a solution would even **cover more use cases** than just jumping to the top of the list _(hence it would close ~3 issues)_ and it would **work for all tabs** (even at the "Updates" tab, where the floating button is used for "update everything"). ...and not to mention it: at all other tabs we might even spare us the annoying floating button completely.

every other attempt to somehow improve or fix the floating button (especially if it takes more than a few minutes) is more or less wasted time IMHO.

**Comment by Iamlooker** [comment]
Closing this as not planed
